src/features/product/ProductCard.tsx

```diff
@@ -0,0 +1,75 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import Box from '@mui/material/Box';
+import { CardActions } from '@mui/material';
+import AddShoppingCartRoundedIcon from '@mui/icons-material/AddShoppingCartRounded';
+import type { ProductConfig } from '../../types/product.ts';
+import ShelterSticker from '../../components/StickerCreator/ShelterSticker.tsx';
+import styles from './ProductCard.module.scss';
+
+const priceSymbol: string = '€';
+
+const handleAddToCart = (event: React.MouseEvent, product: ProductConfig) => {
+  //todo: implement logic of adding sticker to cart
+  event.stopPropagation();
+  console.log('Added to cart:', product);
+};
+
+export default function ProductCard({ product }: { product: ProductConfig }) {
+  return (
+    <Card
+      className={styles.card}
+      sx={{
+        borderRadius: '36px',
+        transition: 'all 0.3s',
+        '&:hover': {
+          transform: 'scale(1.01)',
```

Comment:
I think that scale is not the best option here, as it makes text lines in description to jump. I sugest to do like this:
```suggestion
          transform: 'translateY(-2%)',
```